KanbanGuidelines.md

```diff
@@ -0,0 +1,49 @@
+# Kanban Guidelines
+
+## Kanban General
+
+A kanban board helps us *visualising work*, *limiting the amount of work in progress*, and *pulling work through the system*.
+The following guidelines will help us standardize our use with Kanban boards, so that we can switch projects efficiently without having to learn a new Kanban method this particular project employs.
+
+## Board layout
+
+We work with the following columns on the board:
+
+1. To Do
+2. In Progress
+3. Review
+4. Test _(Optional)_
+5. Done
+
+(image to be added)
+
+_Figure 1 - Kanban board example with all columns and work items_
+
+## Board rules
+
+The team objective is to move work items on the board to the _Done_ lane as fast and best as we can. Note that this implies focus on incremental progress of the application we are building.
+
+- You *should* only have work item on your name.
+_Strive for focus and avoid multitasking._
+- You *should* take the top work item from the To Do column when starting a new item.
+_Work items in To Do *should* be prioritized in descending order._
+- After finishing a work item from any lane, drag it to a new lane, and change it to 'Unassigned'.
```

Comment:
The main takeaway here is that you should not assign tickets to people because that would lead to uncertainty about what this person is actually working on. We state that we want to avoid multitasking, and instead rather focus on a single task for better performance. Having one ticket assigned to your name fits this intent, and also communicates clearly to team members what you're working on. 
I will update the ticket to communicate this more clearly.